docs/path.md

```diff
@@ -0,0 +1,45 @@
+# Allow Path and Ignore Path
+
+## Description
+
+Paths that are not provided by `--allow-path` will be rejected.
+Paths that are provided by `--ignore-path` won't be rejected.
+It is only possible to define one of both.
+
+## Change
```

Comment:
Should we have a separate section for changes to previous kube-rbac-versions?

Reply:
Do you mean in addition to the CHANGELOG.md?